config/initializers/pagy.rb

```diff
@@ -0,0 +1,6 @@
+require 'pagy/extras/items'
+
+Rails.application.config.assets.paths << Pagy.root.join('javascripts')
```

Comment:
Az assetek include path módosításait a `config/application.rb` -be szoktuk rakni